packages/e2e-tests/specs/performance/front-end-twentytwentythree.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { basename, join } from 'path';
+import { writeFileSync } from 'fs';
+
+/**
+ * WordPress dependencies
+ */
+import { activateTheme, createURL, logout } from '@wordpress/e2e-test-utils';
+
+describe( 'Front End Performance', () => {
+	const results = {
+		timeToFirstByte: [],
+	};
+
+	beforeAll( async () => {
+		await activateTheme( 'twentytwentythree' );
+		await logout();
+	} );
+
+	afterAll( async () => {
+		await activateTheme( 'twentytwentyone' );
+		const resultsFilename = basename( __filename, '.js' ) + '.results.json';
+		writeFileSync(
+			join( __dirname, resultsFilename ),
+			JSON.stringify( results, null, 2 )
+		);
+	} );
+
+	it( 'Time To First Byte (TTFB)', async () => {
+		let i = 10;
+		while ( i-- ) {
+			await page.goto( createURL( '/' ) );
+			const navigationTimingJson = await page.evaluate( () =>
+				JSON.stringify( performance.getEntriesByType( 'navigation' ) )
+			);
+			const [ navigationTiming ] = JSON.parse( navigationTimingJson );
+			results.timeToFirstByte.push(
+				navigationTiming.responseStart - navigationTiming.requestStart
```

Comment:
TTFB, by [definition](https://web.dev/ttfb/), is the time from `startTime` to `responseStart`. I guess we can just do `responseStart` here? Or should we use a different metric or a different name?

Reply:
Update: We should change the name to something like `requestTime`. And maybe also include TTFB additionally if needed.

Reply:
Update: We should change the name to something like `requestTime`. And maybe also include TTFB additionally if needed.

Reply:
https://github.com/WordPress/gutenberg/pull/47037#issuecomment-1404101794